PopulateTools/gobierto
# Refactor pages and news

## :v: What does this PR do?

This PR implements partially #932 

- It simplifies the URLs of pages and news to be:
  - /paginas/`collection-slug`
  - /pagina/`page-slug`
- It uses a template system depending on the collection container:
  - for site containers, pages use `gobierto_cms/pages/templates/`
  - for participation, pages use `gobierto_participation/pages/templates/`
  - for participation processes, pages use `gobierto_participation/processes/pages/templates/`

A lot of methods, templates and actions have been removed too.

## :mag: How should this be manually tested?

Checking this issue will be hard. Here there are a few suggested steps:

- check a collection of pages with a Site as container
- check a collection of pages with Participation as container
- check a collection of pages with a participation process as container
- check a collection of news with a Site as container
- check a collection of news with Participation as container
- check a collection of news with a participation process as container
- check a site where a section is configured as the home page
- check the information page of a process
- check the news page of participation
- check the news page of a process

## :eyes: Screenshots

Doesn't apply, it changes at URL level
